dashboard/db/migrate/20240412134643_create_new_feature_feedbacks.rb

```diff
@@ -0,0 +1,14 @@
+class CreateNewFeatureFeedbacks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :new_feature_feedbacks do |t|
+      t.belongs_to :user, type: :integer, null: false, foreign_key: true
+
+      t.integer :form_key, null: false
+      t.boolean :satisfied, null: false, index: true
+
+      t.index [:user_id, :form_key], unique: true
+
+      t.datetime :created_at, null: false
```

Comment:
This is a more standard way of creating `created_at` and `updated_at` columns.

```suggestion
      t.timestamps
```